hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieLogFile.java

```diff
@@ -96,7 +96,11 @@ public FileStatus getFileStatus() {
     return fileStatus;
   }
 
-  public HoodieLogFile rollOver(FileSystem fs) throws IOException {
+  public void setFileStatus(FileStatus fileStatus) {
+    this.fileStatus = fileStatus;
+  }
+
+  public HoodieLogFile rollOver(FileSystem fs, String logWriteToken) throws IOException {
```

Comment:
If you can obtain logWriteToken from log file, do we use the same one when rolling over? would n't that cause two concurrent writers to use the same one?

Reply:
Very good catch. This is a bug which could cause the ingestion job to fail. I have fixed it by tracking  rolloverLogWriteToken and using it for any new log-file creation